test/Api.Tests/TestStartup.cs

```diff
@@ -14,7 +12,9 @@ public void ConfigureTestServices(IServiceCollection services)
             base.ConfigureServices(services);
 
             // Register here the dependency injection services
-           // services.AddDbContext<ApplicationDbContext>(opts => opts.UseInMemoryDatabase());
```

Comment:
Dammed! Stop annoying the developers of the framework. The problem isn't the framwork -.-